share/github-backup-utils/ghe-gc-disable

```diff
@@ -9,8 +9,8 @@
 set -e
 
 # Bring in the backup configuration
-cd $(dirname "$0")/../..
-. share/github-backup-utils/ghe-backup-config
+export GHE_BACKUP_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
```

Comment:
I did try and avoid changing ghe-backup-config behavior but ran in to a
issue with it requiring the working dir be changed to the root backup utils
dir before being called.

In share/github-backup-utils/ghe-gc-disable
<https://github.com/github/backup-utils/pull/179#discussion_r57606669>:

> @@ -9,8 +9,8 @@
>  set -e
>
>  # Bring in the backup configuration
> -cd $(dirname "$0")/../..
> -. share/github-backup-utils/ghe-backup-config
> +export GHE_BACKUP_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"

I mean avoid the export and don't use the GHE_BACKUP_ROOT variable name for
now. That'd change ghe-backup-config behaviour I think.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/github/backup-utils/pull/179/files/26feaa1f4392b21b825040e0c268ecf1cb0fd5da..3427dfe7df2867bd54250b37ee7c4ae7e150efef#r57606669>